repo_health/index/static/client/src/components/issue-stats/issue-stats.controller.js

```diff
@@ -0,0 +1,22 @@
+/*
+*issue-stats.controller.js - (C) Copyright - 2017
+* This software is copyrighted to contributors listed in CONTRIBUTIONS.md.
+*
+* SPDX-License-Identifier: MIT
+*
+* Author(s) of this file:
+* J.Harding
+*
+* Controller for the issue stats named view.
+*
+*/
+
+import BaseStatsController from '../base-component/base-stats.controller';
+
+export default class IssueStatsController extends BaseStatsController {
```

Comment:
Yeah same here, extending with angular is odd. I would have the BaseController be on the first page of our app and do like a `$onChanges`(checking if the urls exist) to change the state/pass down the component tree.